app/src/main/kotlin/app/youkai/data/local/Credentials.kt

```diff
@@ -40,4 +40,12 @@ object Credentials {
 
     val isAuthenticated: Boolean
         get() = username != null && authToken != null
-}
\ No newline at end of file
+
+    fun logout() {
+        authToken = null
+        username = null
+        refreshToken = null
+        userId = null
```

Comment:
Should we go with `prefs.edit().clear()` instead here? 🤔

Reply:
✔️